src/i18n.js

```diff
@@ -0,0 +1,23 @@
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    detection: {
+      order: ['localStorage', 'cookie', 'navigator', 'htmlTag', 'querystring'],
+    },
+    whitelist: ['pl', 'en'],
+    fallbackLng: 'pl',
+    debug: true,
+
```

Comment:
Can you try addin here: `lng: 'pl', // default language` ?

Reply:
If that doesnt work, try other solutions from here: https://stackoverflow.com/questions/65921459/how-to-set-i18n-default-translation-language-in-react

Reply:
Some more docs about config: https://i18next.github.io/i18next/pages/doc_init.html